src/Rules/StarterWeb/AI/IndividualAuth/project.json

```diff
@@ -58,6 +53,13 @@
       "version": "__AspNetCorePlatformVersion__",
       "imports": "portable-net45+win8+dnxcore50"
     },
+    "Microsoft.EntityFrameworkCore.Tools": {
+      "version": "__AspNetCorePlatformVersion__",
+      "imports": [
+        "portable-net45+win8+dnxcore50",
+        "portable-net45+win8"
```

Comment:
net45+win8+dnxcore50 alone did not work (this is the one that is required to get JSON.NET imported without it trying to pull in System.Runtime 2.0.5.0

net45+win8 is required for IX-Async and Remotion since they don't support dnxcore50